.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Was this addition intentional?

Reply:
we use golang as IDE, which by default will generate some metadata in the project path.  this one is for excluding those metadata